app/models/randomizer.rb

```diff
@@ -2,6 +2,7 @@ class Randomizer < ApplicationRecord
   include Raceable
 
   belongs_to :game
+  has_many_attached :files
```

Comment:
How come you went with `has_many` and not `has_one`?  (I know nothing about active storage)

Reply:
Mostly because there's really no downside to `has_many`, and I figured some people/games might need to share multiple files (though I don't know of any). If there's a downside I haven't thought of, open to changing.

Reply:
Ah, good idea I didn't even think of needing to share multiple files.